web/styles/app_components.css

```diff
@@ -1054,6 +1054,16 @@ div.overlay {
                 word-wrap: unset;
             }
 
+            .copy-overlay-message {
+                cursor: pointer;
+                color: hsl(0deg 0% 54%);
+                opacity: 0.7;
```

Comment:
I think we should get out of the habit of specifying opacity separately from the color, since we can express this as `hsl(0deg 0% 54% / 70%)`. Then we can replicate the color on the hover state.